### PR TITLE
ENCD-4639 Organize demo cloud yaml

### PR DESCRIPTION
**Pull request by caseylitton**

### CODE REVIEW

File: cloud-build/run-script.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```
Comment (caseylitton) [inline cloud-build/run-script.sh]: Testing only, not to be checked in

File: cloud-config.yml [hunk 2]
```diff
@@ -96,12 +96,26 @@ bootcmd:
 - cloud-init-per once mkswap-swapfile mkswap /swapfile
```
Comment (hitz) [inline cloud-config.yml]: Should probably cross check with @lrowe 's https://github.com/ENCODE-DCC/encoded/pull/2652 for python 3.7/ES6+

Comment (caseylitton) [reply cloud-config.yml]: This should only change the cloud-config yaml.  lrowes didn't have any config/deploy like changes.

File: cloud-build/post-install-install.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+a2enmod ssl
+a2ensite encoded.conf
+a2dissite 000-default
+a2enconf logging
+a2disconf charset
+a2disconf security
+a2disconf localized-error-pages
+a2disconf other-vhosts-access-log
+a2disconf serve-cgi-bin
+if test "%(ROLE)s" = "demo"
```
Comment (hitz) [inline cloud-build/post-install-install.sh]: I wonder, given our experience with ENCD-4613 if the batchupgrade should be broken out.  I am 99% sure the order doesn't matter, just that it's supposed to run before the machine reboots with these apache config changes.

Comment (caseylitton) [reply cloud-build/post-install-install.sh]: Yea, that's a good idea.  If it's easy I'll through that in.

**Pull request merged**